### ISSUE TITLE
Address the time concern from users

### ISSUE THREAD

**Issue by sstrassberg** [usds/SME-QA-Website#271]
**Legend**
**U**: Usability (Major, Minor, Cosmetic)
**C**: Complexity (High, Medium, Low)
**P**: Priority (Now, Next, Later)

U: Cosmetic; C: Low; P: Next

- [ ] Add updated graphic from github to website (https://github.com/usds/SME-QA-Website/blob/master/_mocks/timeline-v5.1%402x.png)
 - [ ] Get feedback from users

**Comment by humancompanion-usds** [comment]
@sstrassberg - There is no key or legend for the icons added to the graphic. Can one be added? I can't really tell from looking what each one means.

**Comment by sstrassberg** [comment]
@humancompanion-usds I can go back to add them. The Hand holding a person is HR; the person talking with the star in the bubble is a SME, and the person with a gear icon is a HM. But maybe these are two complex and should be simplified?

**Comment by sstrassberg** [comment]
Overtaken by events. Another graphic was generated.
